pages/toast/toast.js

```diff
@@ -0,0 +1,14 @@
+const openToast = document.querySelector('#open-toast');
+const toast = document.querySelector('.toast')
+
+
+const clearToast = (timeout)=>{
+  toast.classList.remove('show');
+  clearTimeout(timeout)
+}
+openToast.addEventListener('click',()=>{
+  console.log('sike')
```

Comment:
Remove console logs from code.

Reply:
Thank you for pointing out the mistake Amar.